src/java.logging/share/classes/java/util/logging/LogManager.java

```diff
@@ -2666,8 +2666,6 @@ private void invokeConfigurationListeners() {
         for (Runnable c : listeners.values().toArray(new Runnable[0])) {
             try {
                 c.run();
-            } catch (ThreadDeath death) {
-                throw death;
```

Comment:
In theory, before this change, some (odd/weird) application code which was registered as a `listener` could throw a `ThreadDeath` and we would then abort any other `listeners` from being run. With this change that would no longer be that case. However, I think that is fine, since the API doc of `LogManager.addConfigurationListener()` already states:

```
* It is recommended that listeners do not throw errors or exceptions.
*
* If a listener terminates with an uncaught error or exception then
* the first exception will be propagated to the caller of
* {@link #readConfiguration()} (or {@link #readConfiguration(java.io.InputStream)})
* after all listeners have been invoked.
```